src/dotnet/commands/dotnet-test/TestRunners/TestRunner.cs

```diff
@@ -51,9 +51,10 @@ private ICommand CreateTestRunnerCommand()
             var commandArgs = _argumentsBuilder.BuildArguments();
 
             return _commandFactory.Create(
-                _testRunner,
+                $"dotnet-{_testRunner}",
                 commandArgs,
-                new NuGetFramework("DNXCore", Version.Parse("5.0")));
+                new NuGetFramework("DNXCore", Version.Parse("5.0")),
+                Constants.DefaultConfiguration);
```

Comment:
We never had that behavior. We were creating the command without passing this param, which was ending up causing it to use Constants.DefaultConfiguration. As part of https://github.com/dotnet/cli/issues/1376, we will actually add the capability to specify a config to it.